node.js/week3/homework/warmup/api/calculator-router.js

```diff
@@ -0,0 +1,212 @@
+const express = require("express")
+const router = express.Router()
+const bodyParser = require("body-parser");
+router.use(bodyParser.urlencoded({ extended: true }));
+
+// Parse URL-encoded bodies (as sent by HTML forms)
+router.use(express.urlencoded({ extended: true }));
+// Parse JSON bodies (as sent by API clients)
+router.use(express.json());
+
+// Addition
+// Getting data through query parameters using GET
+router.get("/Add/", async (request, response) => {
+    try {
+        const firstParam = request.query.firstParam
+        const secondParam = request.query.secondParam
+        let Addition = 0
+        for (i in request.query) {
+            if (!Number.isInteger(parseInt(request.query[i]))) {
+                response.statusCode = 400;
+                response.json({ "message": "bad input" });
+                return;
+            } else {
+                Addition += request.query[i]
+            }
+        }
+        if(firstParam != null && secondParam != null){
+            response.send({ Addition })
+        } else {
+            response.send("Please enter at leats two numbers")
+        }
+    } catch (error) {
+        throw error
+    }
+})
+
+// Subtraction
+// Getting data through query parameters using GET
+router.get("/Subtract/", async (request, response) => {
+    try {
+        const firstParam = request.query.firstParam
+        const secondParam = request.query.secondParam
+        let Subtraction = firstParam * 2
```

Comment:
Why did you multiply firstParam by 2?

Reply:
because at the first iteration on line 50 I will decrease it again